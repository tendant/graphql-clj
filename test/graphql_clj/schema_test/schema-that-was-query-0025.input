input Hello {
    world: String!
    this: Int
   }