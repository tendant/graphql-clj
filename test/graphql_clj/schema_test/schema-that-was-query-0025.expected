{:input-objects
 {:Hello
  {:fields {:world {:type (non-null String)}, :this {:type Int}}}}}
